### PR TITLE
Sp 92 UI

### PR DESCRIPTION
**Pull request by c0rnkrump3t**

### CODE REVIEW

**Review by guaj — CHANGES_REQUESTED** [review]

File: src/components/Calendar/Event/EventDetails.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                                />
+                            </div>
+                        </Grid>
+
+                        {/** Event Start Date */}
+                        <Grid item xs={12} md={12}>
+
+                            <div style={{ paddingTop: '10px', paddingBottom: '10px' }}>
+                                <LocalizationProvider dateAdapter={AdapterDayjs}>
+                                    <CalendarTextField
```
[the remark this answers is not in the document]
Comment (guaj) [reply src/components/Calendar/Event/EventDetails.js:110]: also create event has duplicates. You can create an eventForm in the custom mui components and call this form from all 3 components. Will reduce code duplication a lot. Also will be easier to test / maintain

**Comment by guaj** [comment]
Also do not forget to post the screenshot of the npm run e2e results.
That way we make sure that you did not break any feature.

File: src/components/Calendar/Event/EventDetails.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (c0rnkrump3t) [reply src/components/Calendar/Event/EventDetails.js:110]: image
image

**Review by guaj — APPROVED** [review]

**Review by mnashat — CHANGES_REQUESTED** [review]
- For the event details page shown below:
image
, it would be better if you edited your `Starting date` formatting to only show the date (i.e. remove 'T00:30:57.000Z' from the display).

- For the drawer shown below:
image
, this becomes redundant when we also have an event details page. We should either have a drawer for the events on the calendar page or the event details page, not both.

- The delete functionality does not work. This is a backend issue, however, so @guaj please fix. The issue is with variable names. In the `deleteOne(...)` method in the EventRepository shown below: 
image
, `eventID: eventID.toString()` should be `_id: eventID.toString()` since we removed the eventID field from the tables.

- You are using `SP-14/Refactor-Calendar-Events` backend branch. However, this branch seems to already be merged into `main`, yet the event details page does not load properly using the `main` backend branch. This warrants further investigation, as compatibility with the `main` backend branch is broken.

Note that I haven't reviewed the code itself, as there is some work to be done. I will review the code once everything is properly integrated.

**Review by mnashat — APPROVED** [review]

**Pull request merged**